### DISCUSSION TITLE
Not working with 115.8.1?

### DISCUSSION THREAD

**Discussion by Zeromoon95** [dngray/ghacks-thunderbird-user.js#47] [General]
I have used user.js for a while and after the latest version is out I starting to have connection issues like imap stuff. I deleted my settings and reinstall thunderbird and it doesn't seem to fix anything. Even changing the autoconfig settings I cannot get to log in into the server, like "Unable to log in at server. Probably wrong configuration, username or password". I'm not sure if this is a compatibility issue. I'm using gmail and yahoo emails.

**Comment by HorlogeSkynet** [comment]
Hello @Zeromoon95, from what I read on TB [115.8.0 release notes](https://www.thunderbird.net/en-US/thunderbird/115.8.0/releasenotes/), it appears OAuth2 code has changed. It is not unlikely autoconfig code also changed.

From [source code](https://searchfox.org/comm-esr115/source/mail/components/accountcreation/ConfigVerifier.jsm#177-202), you're unfortunately experiencing the "generic" error message.

Would it be possible for you to downgrade TB, for instance to 115.7.0, to nail down the issue to the latest minor version ?

You can also :

* re-enable developer tools (`devtools.chrome.enabled`) and open the debugging console to check whether you got JavaScript errors when trying to login ;
* enable or change log level (see <https://wiki.mozilla.org/MailNews:Logging>).

Thanks, bye :wave:

**Comment by HorlogeSkynet** [comment]
Any news @Zeromoon95 ?

**Reply by Zeromoon95** [reply]
I just use the non-user.js version and waiting until the next version drops.

I don't think changing java settings can work and the log level is very complicated.

**Reply by HorlogeSkynet** [reply]
Hello, any chance it has been fixed with 115.9.0 ? :upside_down_face:

**Comment by HorlogeSkynet** [comment]
Closing here due to stale, please do re-open if that's not the case.
